### PR TITLE
zziplib: Make library names same as pkgconfig files.

### PR DESCRIPTION
**Pull request by Biswa96**
Revert+fix of https://github.com/msys2/MINGW-packages/pull/8545 and https://github.com/msys2/MINGW-packages/pull/8657

Fixes https://github.com/msys2/MINGW-packages/issues/8742

### CODE REVIEW

File: mingw-w64-zziplib/PKGBUILD [hunk 1]
```diff
@@ -4,61 +4,83 @@ _realname=zziplib
...
 }
 
 package() {
-  cd "${srcdir}/build-${MINGW_CHOST}"
-  DESTDIR="${pkgdir}" ninja install
+  cd "${srcdir}/build-${CARCH}-static"
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --build ./ --target install
+
+  cd "${srcdir}/build-${CARCH}-shared"
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --build ./ --target install
```
[the remark this answers is not in the document]
Comment (Biswa96) [reply mingw-w64-zziplib/PKGBUILD:82]: AFAIK, shared libs are preferred here. So, shared are installed later to replace any cmake or other files from static ones. At least, the idea has been working in other packages also.

[the remark this answers is not in the document]
Comment (mati865) [reply mingw-w64-zziplib/PKGBUILD:82]: The thing is BFD and LLD can link symbols from shared libraries when headers do not have `dllimport` (like ones from static libraries) but they cannot link symbols from static libraries when headers have `dllimport`.
So basically static headers work for both shared and static while shared headers work only for shared linking.

[the remark this answers is not in the document]
Comment (Biswa96) [reply mingw-w64-zziplib/PKGBUILD:82]: Understand. But here the focus is on fixing deps. If we consider that header thing we need to fix many packages for static linking.

**Comment by Biswa96** [comment]
Ping!

**Comment by Biswa96** [comment]
OK?

**Pull request merged**